rca/project_styleguide/templates/patterns/molecules/search-result/search-result.html

```diff
@@ -6,9 +6,13 @@
             <h3 class="search-result__heading heading heading--five">{% firstof result.listing_title result.title %}</h3>
             {% if search_pick %}<div class="search-result__editor-pick body body--support">Recommended result</div>{% endif %}
         </div>
-        {% if result.listing_meta %}<p class="search-result__meta layout__@large-start-three layout__@large-span-one body body--two">{{ result.listing_meta }}</p>{% endif %}
-        {% if pick.description or result.listing_summary or result.search_description %}
-            <p class="search-result__summary layout__@large-start-four layout__span-two body body--one">{% firstof pick.description result.listing_summary result.search_description %}</p>
+        {% with meta=result.specific.listing_meta %}
+            {% if meta %}
+                <p class="search-result__meta layout__@large-start-three layout__@large-span-one body body--two">{{ meta }}</p>
+            {% endif %}
+        {% endwith %}
```

Comment:
@richbrennan I can't quite figure out a way of showing the `listing_meta` values (defined as a property on the models) [eg](https://github.com/torchbox/rca-wagtail-2019/pull/755/files#diff-30730dc47e8d4f1f4999a400ad6d16177570579e2b38573070910a7dfe691bbcR265) without using specific here. Any thoughts?

Reply:
Is it possible to call `.specific()` on the queryset before it gets passed to the template?